install/helm/openchoreo/templates/asgardeo-thunder/post-install-hook.yaml

```diff
@@ -87,10 +87,7 @@ spec:
                       "response_types": [
                         "code"
                       ],
-                      "token_endpoint_auth_methods": [
```

Comment:
Have we checked whether https://github.com/openchoreo/openchoreo/tree/main/install/helm/openchoreo-control-plane chart is also updated?